### PR TITLE
Use docsify to make the documentation pretty

### PR DESCRIPTION
**Pull request by botnerd**
This uses [Docsify](https://docsify.js.org/#/) to generate a pretty website for documentation. Note that this is dynamically generated, not static html. This restructures the documentation so it is better organized and iOS/Android SDK documentation have been combined with tabs to switch between code samples for each platform.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![FlyBuy Documentation](readme.png)
+# FlyBuy Developer Documentation {docsify-ignore-all}
```
Comment (jnebeker) [inline README.md:1]: Is there a hidden way to tell docsify to ignore this file? If anyone is reading the README for this repo they're now greeted with a big "{docsify-ignore-all}".

File: sdk/customer.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            ageVerification = userVerifiedAge) { customer, sdkError ->
+        // Handle customer or deal with error
+    }
+}
+```
+<!-- tabs:end -->
+
+?> **Note** The `phone` parameter is optional. Set it to an empty string (`""`) if not use.
+
+!> **Important** If the `termsOfService` and `ageVerification` parameters are not `true`, an error will be returned and the customer will not be created.
```
Comment (jnebeker) [inline sdk/customer.md:57]: I love the look of these sections in the docs 🔥

**Comment by jwieringa** [comment]
For cross reference, @botnerd setup a call with me and @csexton. We now have a plan on how we'll deploy.

Follow along here: https://github.com/RadiusNetworks/cyclops/issues/249

**Comment by botnerd** [comment]
> Love it! ❤️ 👏 🙌
> 
> I'll defer to @csexton about next steps for getting this released, I know we'd need to rework the doc publishing.
> 
> Any idea if we can get rid of [these `<span>` tags](https://github.com/RadiusNetworks/flybuy-documentation/blob/im-so-pretty-oh-so-pretty/api/v1/reporting.md#get-a-list-of-all-archived-orders) after we make this switch? They're ruining the beautiful search bar. 😭
> 
> image

I didn't edit this file because I didn't know if it was one of the ones generated by the code. It's easy enough to remove.

**Comment by botnerd** [comment]
Closing as this will be published from another repo.

**Pull request closed without merging**